scripts/beck/st/error_injection.py

```diff
@@ -295,62 +335,77 @@ def induce512kbError():
 				if j > 524288:
 					break
 				else:
-					wbfr.seek(offset)
-					wbfr.write(b'\x33\x33\x44\x44\x22\x22\x11\x11')
-					#ReadMetaData(offset)
+					EditMetadata(offset)
+					#wbfr.seek(offset)
+					#wbfr.write(b'\x33\x33\x44\x44\x22\x22\x11\x11')
+					count = count + 1
+	return count
 
 		
 def verifyLengthOfRecord(recordDict):
-	print("***********Record list will be print here************")
+	count = 0
+	read_metadata_file()
+	logger.info("***********Record list will be logger.info here************")
 	for record,items in recordDict.items():
-		print(" {} :  {}".format(record, len(items)))
-
-	
-if args.err512k:
-	induce512kbError()
-	
-if args.hugeCorruption:
-	induceHugeError()
+		logger.info(" {} :  {}".format(record, len(items)))
+		count = count + 1
+	return count
```

Comment:
Where is the "verify"?
It only output the records into logs.
It does not "verify".

Reply:
Yes, that true. It will print out all record numbers in the log So user can verify those number with a dry run of beck tool.